website/docs/general/documentation-guide.md

```diff
@@ -0,0 +1,129 @@
+- Write for accessibility. See [General do's and don'ts](https://developers.google.com/style/accessibility#general-dos-and-donts) to learn more.
+- Identify the intended audience and write for them. Find the common ground for you and your target audience.
+
+## Language
+
+- Use second person while writing unless it isn't appropriate.
+- Use active voice while writing.
+- Use standard American spellings.
+- Always place [conditional clauses before instructions](https://developers.google.com/style/clause-order).
+- Use the following spellings for project specific entities:
+  - Use Apache APISIX instead of APISIX when referencing the project in introductions and when referencing to the project community.
+  - Use APISIX instead when referencing to the project inside documentation.
+  - APISIX specific component names like Plugin and Route are always capitalized.
+  - Use correct acronyms when necessary. Use the expanded word in first usage and proceed to use the acronym in the rest of the references. For example:
+
+    | ✅               | ❌               |
+    | ---------------- | ---------------- |
+    | URL              | url              |
+    | API              | api              |
+    | APISIX Dashboard | Apisix dashboard |
+    | gRPC             | GRPC/grpc        |
+
+## Formatting, punctuation and organization
+
+- Use sentence case for all headings and subheadings.
+- Use advanced markdown features and admonitions provided by Docusaurus when necessary:
+  - Use [tabs](https://docusaurus.io/docs/next/markdown-features/tabs) and [synced tabs](https://docusaurus.io/docs/next/markdown-features/tabs#syncing-tab-choices) when you have to show multiple paths based on a user's configuration or environment.
+  - Use [code blocks](https://docusaurus.io/docs/next/markdown-features/code-blocks) to display code. Add the file being modified as the title of the codeblock (for example, `coonf/config.yaml`). Use line highlighting to focus the attention of the use while explaining code.
+  - Use [admonitions](https://docusaurus.io/docs/next/markdown-features/admonitions) to highlight important information. Use them as suggested below:
+    - Note: for general information you want to stand out.
+    - Tip: to give users bonus tips that build on top of the rest of the content. Can be ignored generally.
+    - Important: to give extra emphasis to general information.
+    - Warning: to highlight possibilities of loss of configuration or data.
+    - Danger: user should be very careful about what is mentioned in the block. Use only in cases of maximum severity.
+- Use tables to convey appropriate information easily. See [how they are used in FAQ](/docs/apisix/FAQ#what-is-the-difference-between-plugin-metadata-and-plugin-configs-in-apache-apisix).
+- Use relative paths within markdown files `../xxx/xxx` instead of absolute paths.
+
+### Directory structure
```

Comment:
It could come under organisation. It shows how the documents are organised and how that organisation is configured in the configuration file. What do you think?